--- a/configure
+++ b/configure
@@ -7281,7 +7281,7 @@
   withval=$with_xulrunner_sdk;
 	XULRUNNER_SDK_PREFIX=$with_xulrunner_sdk
 	GECKO_CFLAGS="-I$XULRUNNER_SDK_PREFIX/include -I$XULRUNNER_SDK_PREFIX/include/plugin -I$XULRUNNER_SDK_PREFIX/include/nspr -I$XULRUNNER_SDK_PREFIX/include/xpcom -I$XULRUNNER_SDK_PREFIX/include/string -I$XULRUNNER_SDK_PREFIX/include/java"
-	GECKO_LIBS="-L$XULRUNNER_SDK_PREFIX/lib -lxpcomglue_s -lxul -lxpcom -lplds4 -lplc4 -lnspr4 -lpthread -ldl"
+	GECKO_LIBS="-L$XULRUNNER_SDK_PREFIX/lib -lxpcomglue_s_nomozalloc -lxul -lxpcom -lplds4 -lplc4 -lnspr4 -lpthread -ldl"
 	# Check whether --enable-new-libxul was given.
 if test "${enable_new_libxul+set}" = set; then :
   enableval=$enable_new_libxul; usenewlibxul=$enableval
--- a/configure.in
+++ b/configure.in
@@ -48,7 +48,7 @@
         [
 	XULRUNNER_SDK_PREFIX=$with_xulrunner_sdk
 	GECKO_CFLAGS="-I$XULRUNNER_SDK_PREFIX/include -I$XULRUNNER_SDK_PREFIX/include/plugin -I$XULRUNNER_SDK_PREFIX/include/nspr -I$XULRUNNER_SDK_PREFIX/include/xpcom -I$XULRUNNER_SDK_PREFIX/include/string -I$XULRUNNER_SDK_PREFIX/include/java"
-	GECKO_LIBS="-L$XULRUNNER_SDK_PREFIX/lib -lxpcomglue_s -lxul -lxpcom -lplds4 -lplc4 -lnspr4 -lpthread -ldl"
+	GECKO_LIBS="-L$XULRUNNER_SDK_PREFIX/lib -lxpcomglue_s_nomozalloc -lxul -lxpcom -lplds4 -lplc4 -lnspr4 -lpthread -ldl"
 	AC_ARG_ENABLE(new-libxul,
 		AC_HELP_STRING([--enable-new-libxul],[libxul/xulrunner-sdk is 1.9.2 or higher]),
 		[usenewlibxul=$enableval],[usenewlibxul=no])
